Description: A patch for config.h.in before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Added support for size_t.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/config.h.in
+++ autotrace-0.31.1/config.h.in
@@ -81,5 +81,8 @@
 /* Define to the version of this package. */
 #undef PACKAGE_VERSION
 
+/* Define as printf format-string modifier for size_t */
+#undef SIZET_FORMAT
+
 /* Define to 1 if you have the ANSI C header files. */
 #undef STDC_HEADERS
